fitbenchmarking/parsing/parse_fitbenchmark_data.py

```diff
@@ -61,8 +61,9 @@ def __init__(self, fname):
         self._type = "FitBenchmark"
 
         self._starting_values = None
-        self._start_x = entries['fit_parameters']['StartX']
-        self._end_x = entries['fit_parameters']['EndX']
+        if 'fit_parameters' in entries:
```

Comment:
Thanks for spotting this and I agree it is reasonable to make StartX and EndX optional